### ISSUE TITLE
Improve layout pokemon details

### ISSUE THREAD

**Issue by igihcksn** [igihcksn/mypocketmonsters#14]
This issue is require to improve pokemon details page layout to get better experience

**Comment by yamitrvg12** [comment]
Hey [igihcksn](https://github.com/igihcksn), can you assign me this task so that I can work on it?

**Comment by igihcksn** [comment]
Yes sure @yamitrvg12, after u done with ur changes please put description and also the screenshot before and after, thank you
